src/Symfony/Bundle/FrameworkBundle/Command/SecretsDecryptToLocalCommand.php

```diff
@@ -38,6 +38,7 @@ public function __construct(
     protected function configure(): void
     {
         $this
+            ->addOption('strict', 's', InputOption::VALUE_NONE, 'Returns a non-zero exit code if any errors are encountered')
```

Comment:
What about naming this `--exit`? Strict doesn't tell what it does

Reply:
Agreed, I updated the MR.